### PR TITLE
IsNullFixer - Fix bug when calling without params

### PR DESCRIPTION
**Pull request by SpacePossum**
cc @kalessil

### CODE REVIEW

File: tests/Fixer/LanguageConstruct/IsNullFixerTest.php [hunk 1]
```diff
@@ -153,6 +153,13 @@ public function provideExamples()
                 '<?php null === a(null === a(null === a(null === b())));',
                 '<?php \is_null(a(\is_null(a(\is_null(a(\is_null(b())))))));',
             ),
+            array(
+                '<?php $d= null === ($a)/*=?*/?>',
+                "<?php \$d= is_null(\n(\$a)/*=?*/\n)?>",
+            ),
+            array(
+                '<?php is_null()?>',
```
Comment (SpacePossum) [inline tests/Fixer/LanguageConstruct/IsNullFixerTest.php]: this case was fixed incorrectly

Comment (kalessil) [inline tests/Fixer/LanguageConstruct/IsNullFixerTest.php]: @SpacePossum hmmm, interesting edge-case. Thanks for fixing!

Comment (SpacePossum) [inline tests/Fixer/LanguageConstruct/IsNullFixerTest.php]: you're welcome, thanks for writing the fixer :)

**Comment by keradus** [comment]
Thank you @SpacePossum.
